src/subcommand/server.rs

```diff
@@ -712,6 +717,57 @@ impl Server {
     Ok(Json(hex::encode(metadata)))
   }
 
+  async fn inscription_details(
+    Extension(page_config): Extension<Arc<PageConfig>>,
+    Extension(index): Extension<Arc<Index>>,
+    Path(inscription_id): Path<InscriptionId>,
+  ) -> ServerResult<Response> {
+    let inscription = index
+      .get_inscription_by_id(inscription_id)?
+      .ok_or_not_found(|| format!("inscription {inscription_id}"))?;
+
+    let entry = index
+      .get_inscription_entry(inscription_id)?
+      .ok_or_not_found(|| format!("inscription {inscription_id}"))?;
```

Comment:
We can `.unwrap` here, because we should never have an inscription in the inscription to ID table without a corresponding entry.